src/test/powersoftau_test.cpp

```diff
@@ -254,7 +286,30 @@ TEST(PowersOfTauTests, ReadPowersOfTauOutput)
     std::ifstream in(filename, std::ios_base::binary | std::ios_base::in);
     srs_powersoftau<ppT> pot = powersoftau_load(in, n);
 
+    std::string expect_pot_write;
+    {
+        std::ifstream in_pot(
+            filename, std::ios_base::binary | std::ios_base::in);
+        expect_pot_write = std::string(
+            std::istreambuf_iterator<char>(in_pot),
+            std::istreambuf_iterator<char>());
+    }
+
+    std::string pot_write;
+    {
+        std::ostringstream out;
+        powersoftau_write(out, pot);
+        pot_write = out.str();
+
+        // powersoftau_write creates a dummy hash at the start so that
+        // data looks of a similar format to the mpc output.  Extract
+        // the important part.  (Below, we only do the comparison on
```

Comment:
> the important part.   (Below,

There are two spaces after `.` instead of one. This also applies to several files in the PR.

Reply:
I've changed the instances that I found in the diffs.